### DISCUSSION TITLE
Can't find the supervisor file in Hue folder

### DISCUSSION THREAD

**Discussion by ckargi55** [cloudera/hue#3161] [Questions]
I can't find the file 'supervisor' in Hue folder. According to official documentation it should be in the folder $HUE_HOME/build/env/bin. I am doing my operation in Ubuntu server 22.04. My objective is to send queries to Impala through Hue.

I run following command as it was written in http://cloudera.github.io/hue/latest/administrator/installation/starting/

`
build/env/bin/supervisor
`
then I got "No such file or directory" warning.

I also tried
`
build/env/bin/hue runserver
`

and I got the same "No such file or directory" warning because there are no such files there. What would you suggest to solve this?

**Comment by bjornalm** [comment]
Hi @ckargi55,

thanks for reaching out. Did you run `make apps` as described here?
https://docs.gethue.com/developer/development/

**Reply by ckargi55** [reply]
Hi @bjornalm,

Thanks for your response. I did those steps but there is no "hue" file in ./build/env/bin folder to run the command ` ./build/env/bin/hue runserver  `
